### PR TITLE
Improve buttons

### PR DESCRIPTION
**Pull request by Gnuk**
* Related to #2894

### CODE REVIEW

**Comment by Gnuk** [comment]
I worked with @ascelineboullen for some colors, fonts and button style.

File: src/main/style/atom/button/_button.scss [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
   background-color: $jhlite-button-color-background;
   cursor: pointer;
+  padding: $jhlite-button-padding;
   text-align: center;
+  text-transform: uppercase;
+  line-height: $jhlite-button-line-height;
   color: $jhlite-button-color-text;
+  font-family: $jhlite-global-font-primary-family;
+  font-size: $jhlite-button-font-size;
+  font-weight: bold;
```
Comment (DamnClin) [inline src/main/style/atom/button/_button.scss:27]: Gave it a try and I think it's really better in default weight cc @Gnuk @ascelineboullen what do you think?

defaultWeight
boldWeight

File: src/main/style/token/_colors.scss [hunk 2]
```diff
@@ -23,10 +30,9 @@ $jhlite-global-color-text-disabled: color.change(
   $saturation: 25%,
   $lightness: 60%
 ) !default;
-$jhlite-global-color-fill-secondary: #353d47 !default;
-$jhlite-global-color-text-light: $jhlite-global-color-light !default;
-$jhlite-global-color-link-light: $jhlite-global-color-light !default;
-$jhlite-global-color-link-light-hover: $jhlite-global-color-primary !default;
+$jhlite-global-color-fill-secondary: $jhlite-color-neutral-900 !default; // @deprecated
```
Comment (DamnClin) [inline src/main/style/token/_colors.scss:33]: Created https://github.com/jhipster/jhipster-lite/issues/4763 as follow up

**Comment by DamnClin** [comment]
Merging like that, comments can be taken in dedicated PRs

**Pull request merged**